docs/ibc/overview.md

```diff
@@ -10,131 +10,147 @@ Learn what IBC is, its components and use cases. {synopsis}
-specific message type and a specific counterparty (perhaps storing a specific connectionEnd as part
-of a handshake, or a packet intended to be relayed to a module on the counterparty chain). A relayer
-process monitors for updates to these paths, and will relay messages, by submitting the data stored
-under the path along with a proof to the counterparty chain. The paths that all IBC implementations
-must use for committing IBC messages is defined in
-[ICS-24](https://github.com/cosmos/ics/tree/master/spec/core/ics-024-host-requirements) and the proof
-format that all implementations must be able to produce and verify is defined in this [ICS-23 implementation](https://github.com/confio/ics23).
+communicate, a blockchain commits some state to a specifically defined path that is reserved for a
+specific message type and a specific counterparty. For example, for storing a specific connectionEnd as part
+of a handshake, or a packet intended to be relayed to a module on the counterparty chain. A relayer
+process monitors for updates to these paths and relays messages by submitting the data stored
+under the path along with a proof to the counterparty chain. 
+
+- The paths that all IBC implementations must use for committing IBC messages is defined in
+[ICS-24 Host State Machine Requirements](https://github.com/cosmos/ics/tree/master/spec/core/ics-024-host-requirements). 
+- The proof format that all implementations must be able to produce and verify is defined in [ICS-23 Proofs](https://github.com/confio/ics23) implementation.
 
 ### [Capabilities](https://github.com/cosmos/cosmos-sdk/blob/master/docs/core/ocap.md)
 
 IBC is intended to work in execution environements where modules do not necessarily trust each
-other. Thus IBC must authenticate module actions on ports and channels so that only modules with the
-appropriate permissions can use them. This is accomplished using [dynamic
-capabilities](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-003-dynamic-capability-store.md). Upon binding to a port or
-creating a channel for a module, IBC will return a dynamic capability that the module must claim in
-order to use that port or channel. This prevents other modules from using that port or channel since
-they will not own the appropriate capability.
-
-While the above is useful background information, IBC modules do not need to interact at all with
+other. Thus, IBC must authenticate module actions on ports and channels so that only modules with the
+appropriate permissions can use them. This module action authentication is accomplished using a [dynamic
+capability store](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-003-dynamic-capability-store.md). Upon binding to a port or
+creating a channel for a module, IBC returns a dynamic capability that the module must claim in
+order to use that port or channel. This dynamic capability store prevents other modules from using that port or channel since
```

Comment:
another unclear reference "This"
I guessed that we are talking about "This dynamic capability store" 

But please teach me if I'm off track